subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ivy/ComponentSelectionRulesDependencyResolveIntegTest.groovy

```diff
@@ -399,9 +399,9 @@ Required by:
             }
 
             checkDeps.doLast {
-                def artifacts = configurations.conf.resolvedConfiguration.resolvedArtifacts
+                def artifacts = configurations.conf.getIncoming().artifactView { }.artifacts.artifacts
```

Comment:
I think the empty artifactView is unnecessary here. We can just call `.artifacts.artifacts` on `getIncoming` directly.

Applies elsewhere

Reply:
True, I'll update these usages.